.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - iojs-v2.0.0
-  - iojs
+  - "4.0"
+  - "4.1"
```

Comment:
"4.2" , "5.2" seems fine. Even if we don't want to support LTS, knowing when LTS is broken would be good :)